Test/SearchEngineTests.cs

```diff
@@ -1611,5 +1612,60 @@ public static void TestFileWithNoMs2Scans()
             var ms2Scans = MetaMorpheusTask.GetMs2Scans(fileWithNoMs2Scans, null, new CommonParameters()).OrderBy(b => b.PrecursorMass).ToArray();
             Assert.That(!ms2Scans.Any());
         }
+
+        //This is a code coverage unit test that mostly just checks there are no crashes in specific situations
+        [Test]
+        public static void TestUndefinedProteinGroup()
+        {
```

Comment:
are there supposed to be some Assertions here? like checking returned values match previously specified values?

Reply:
No, just checking that the program keeps moving when things aren't functioning as expected.